### DISCUSSION TITLE
是否”增量预训练“和”精调“调整的都是LoRA参数？

### DISCUSSION THREAD

**Discussion by RedZh** [ymcui/Chinese-LLaMA-Alpaca-2#366] [Q&A] [answered]
如题，
另外精调(得到的Chinese-Alpaca-2)是在调整增量预训练(得到的Chinese-LLaMA-2)的LoRA参数吗？

**Comment by ymcui** [comment, marked as answer]
训练什么参数取决于训练脚本，和“预训练”、“精调”这些词无关。
这里说的“预训练”指的是无标注数据训练，“精调”指的是“指令精调”，即使用指令数据进行有监督的训练。
我们提供的代码主要以lora训练为主。如果你想全量预训练就自己改改代码吧。

**Reply by RedZh** [reply]
感谢回答。我明白提供的代码是LoRA训练，请问发布的Chinese模型(llama2 & alpaca2)也都是LoRA训练后的结果吗？还是全量预训练的结果？

**Reply by ymcui** [reply]
是。论文里有相关资料。FAQ里也有解答（[LINK](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/faq_zh#问题4为什么不对模型做全量预训练而是用lora)）。

**Reply by RedZh** [reply]
好的谢谢
